Source/Controllers/MainViewControllers/SPDatabaseDocument.m

```diff
@@ -417,6 +417,23 @@ - (void)setConnection:(SPMySQLConnection *)theConnection
     // Ensure the connection encoding is set to utf8 for database/table name retrieval
     [mySQLConnection setEncoding:@"utf8mb4"];
 
+    // Check if skip-show-database is set to ON
+    SPMySQLResult *result = [mySQLConnection queryString:@"SHOW VARIABLES LIKE 'skip_show_database'"];
+    [result setReturnDataAsStrings:YES];
+    if(![mySQLConnection queryErrored]) {
+        if ([result numberOfRows] == 1) {
+            NSString *skip_show_database = [[result getRowAsDictionary] objectForKey:@"Value"];
+            if ([skip_show_database caseInsensitiveCompare:@"on"] == NSOrderedSame) {
+                NSAlert *alert = [[NSAlert alloc] init];
```

Comment:
Please use `NSAlert createWarning` or any alternative one that we have in Swift extension :)

Reply:
It's done.